StandardConfig/lcgeo_current/sdhcal_digi.xml

```diff
@@ -0,0 +1,131 @@
+
+ <!--Induced charge dispatching parameters-->
+ <!--Induced charge dispatching : which steps to use parameters-->
+  <!--Maximum distance (mm) between the Geant4 step position and the cell center, in the RPC width direction, to keep a step for digitization. 
+Default value=0.0005-->
+  <parameter name="StepCellCenterMaxDistanceLayerDirection" type="float">0.0005 </parameter>
+  <!--if true, ensure that each hit will keep at least one step for digitisation independatly of filtering conditions (StepCellCenterMaxDistanc
+eLayerDirection)-->
+  <parameter name="KeepAtLeastOneStep" type="bool">true </parameter>
+  <!--Minimum distance (mm) between 2 Geant4 steps, in the RPC plane, to keep the 2 steps. Default value=0.5-->
+  <parameter name="StepsMinDistanceRPCplaneDirection" type="float">1.2 </parameter>
+
+ <!--Induced charge dispatching : dispatching mode-->
+  <!--Define the charge splitter method. Possible option : Uniform, Function, Erf-->
+  <parameter name="ChargeSplitterOption" type="string"> Erf </parameter>
+  <!--If true, use function to split the charge between pads. If false split uniformely-->
+  <!--parameter name="UseFunctionForChargeSplitting" type="bool">true </parameter-->
+
+ <!--Induced charge dispatching : dispatching uniformely parameters-->
+  <!--Distance (in mm) from the border of the cell that implies hit on neighbouring cell for 'Uniform' charge splitter : used if UseFunctionFor
+ChargeSplitting is false-->
+  <parameter name="CellEdgeDistance" type="float">2 </parameter>
+
+ <!--Induced charge dispatching : dispatching with function parameters-->
+  <!--ROOT TF2 formula for function describing the induced charge distribution, x and y are coordinates from the step position in I and J direction: used if UseFunctionForChargeSplitting is true-->
+  <parameter name="functionFormula" type="string"> exp(-(x*x+y*y)/([0]*[0])) +[1]*exp(-(x*x+y*y)/([2]*[2]))</parameter>
+  <!--parameter values for the function defined with functionFormula-->
+  <parameter name="functionParameters" type="FloatVec">1. 0.00067 11.2 0.000005 80</parameter>
+  <!--Width values for the different Erf functions-->
+  <parameter name="erfWidth" type="FloatVec">1. 10.</parameter>
+  <!--Weigth for the different Erf functions-->
+  <parameter name="erfWeigth" type="FloatVec">1. 0.00077 </parameter>
+  <!--distance in mm between two RPC pads : used if UseFunctionForChargeSplitting is true -->
+  <parameter name="RPC_PadSeparation" type="float">0.408 </parameter>
+  <!--maximal distance (in mm) at which a step can induce charge using the 2D function defined with functionFormula-->
+  <parameter name="functionRange" type="float">30 </parameter>
+
+ <!--digitisation parameters--> 
+  <!--Threshold for HCAL Hits in pC-->
+  <parameter name="HCALThreshold" type="FloatVec">0.114 5.4 14. </parameter>
+  <!--parameter name="HCALThreshold" type="FloatVec">0.114 1.0 15.5 </parameter-->
+  <!--Calibration coefficients for HCAL-->
+  <!-- DBD parameters -->
```

Comment:
Can you please tidy up this section, removing un-needed xml comments and keep only one CalibrHCAL parameter